docs/reference/events.md

```diff
@@ -61,6 +109,33 @@ This event is fired **after** a part has been released. Raising an exception thu
 |cdbprot_remark|str \| None | Remark|
 |cdb_ec_id|str \| None| Engineering Change ID|
 
+## PartReleaseCheckEvent
+`csfunctions.events.PartReleaseCheckEvent`
+
+This event is fired when a part is about to be released, but **before** the release has happened. Raising an exception will prevent the release.
+At this point it is not guaranteed yet that the part will be released, meaning the part should not be sent to e.g. an ERP system.
```

Comment:
Not clear to me. Why is it not guaranteed? Because you code can send an abort-action? Or because any other error could occur? And release does not mean that it 

Suggestion:
> At this point the part is not yet released. You should not use this Event to sent the part to e.g. an ERP system.

Reply:
Yes. Any error could still prevent the release. I will clarify this part in the docs.

Reply:
[f88ae56](https://github.com/cslab/functions-sdk-python/pull/13/commits/f88ae56d4997cd132477c85cc8a268b713aff046)